src/styles/index.scss

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.stopwatch {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dashed #000;
+  border-radius: 50%;
+  border-style: dotted;
+
+  &__minute {
+    animation: mintime 3600s steps(60, end) infinite;
+    position: absolute;
+    transform: translate(-50%, -100%);
+    margin-top: 20vmin;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+  }
+
+  &__second {
+    animation: sectime 60s linear infinite;
+    position: absolute;
+    transform: translate(-50%, -100%);
+    margin-top: 2vmin;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+  }
+
+  &__center {
+    position: absolute;
+
+    z-index: 1;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 50%;
+    background-color: #f6a603;
+  }
+
+  &--speed-up {
+    &__minute {
+      animation-duration: 600s;
+    }
+
+    &__second {
+      animation-duration: 10s;
+    }
+  }
+}
+
+@keyframes sectime {
+  from {
+    transform: rotate(0deg);
+    transform-origin: center bottom;
+    transition-duration: 60s;
+    transition-timing-function: linear;
```

Comment:
The `transition-duration` and `transition-timing-function` properties are not effective within keyframes. Transitions do not apply to keyframe animations. Consider removing these properties from the keyframes.